src/pages/index.js

```diff
@@ -0,0 +1,304 @@
+import "./index.css";
+import {
+  enableValidation,
+  settings,
+  resetValidation,
+  disabledButton,
+} from "../scripts/validation.js";
+import Api from "../utils/api.js";
+
+const api = new Api({
+  baseUrl: "https://around-api.en.tripleten-services.com/v1",
+  headers: {
+    authorization: "3b2e2121-8c57-411a-855f-0d4b0fcfc9ae",
+    "Content-Type": "application/json",
+  },
+});
+
+api
+  .getAppInfo()
+  .then(([cards, user]) => {
+    cards.forEach((item) => {
+      const cardElement = getCardElement(item);
+      cardsList.append(cardElement);
+    });
+    document.getElementById("user-name").textContent = user.name;
+    document.getElementById("user-about").textContent = user.about;
+    document.getElementById("user-avatar").src = user.avatar;
```

Comment:
`document.getElementById("user-name")`
`document.getElementById("user-about")`
`document.getElementById("user-avatar")`


It’s better to find **constant DOM elements** only 1 time at the top of the file so as not to waste resources on searching them again and again when you call a method (function) because searching is a very **hard** operation for the browser engine


You have `profileName`, `profileDescription` for that